internal/pkg/service/common/httpserver/middleware/otel.go

```diff
@@ -55,14 +46,7 @@ func OpenTelemetry(tp trace.TracerProvider, mp metric.MeterProvider, cfg Config)
 			ctx = context.WithValue(ctx, redactedRouteParamsCtxKey, cfg.redactedRouteParams)
```

Comment:
I see no change in tests, so there no issue, right?

Maybe your are talking about `otelhttp.LabelerFromContext`, the attributes are generated by the `OpenTelemetry` middleware, by the `otelhttp.Handler`.


It would be nice to have 2 tests, for OTEL middleware (basic metrics), and for OTEL+ApDex middlewares (all metrics).

Reply:
Yes, then can be test for 1. `OTEL`, and 2. `OTEL+apdex`. 

`apdex only` should also work, but those attributes don't appear, as you wrote.